lib/parsers.js

```diff
@@ -252,3 +280,23 @@ function tryJsonStringify (obj) {
     return JSON.stringify(obj)
   } catch (e) {}
 }
+
+function getRelativeFileName (filename) {
+  var root = process.cwd()
+  if (root[root.length - 1] !== path.sep) root += path.sep
+  return !~filename.indexOf(root) ? filename : filename.substr(root.length)
+}
+
+/**
+ * Below two functions expects the passed in object to be of type stackframe
```

Comment:
Multiline comments should be written with regular `//`. I'd also be careful with one comment concerting two functions as things might get refactored. In cases like this, I think it's better to just repeat the comment above each function 🙂